CorsixTH/Lua/dialogs/message.lua

```diff
@@ -105,6 +105,11 @@ function UIMessage:adjustToggle()
 end
 
 function UIMessage:openMessage()
+  if TheApp.world:checkPauseBehaviour() and not self.ui:checkForMustPauses() then
+    self.ui:playSound("wrong2.wav")
+    self:adjustToggle()
+    return
+  end
```

Comment:
The topic says you want to block opening faxes, so this spot is too late judging from the function name.
At this point the window already exists and the decision to open it has already been made (it seems to be called from the window being opened).

Blocking in my view means not even attempt to create or open it.